conf/log4j2.properties

```
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+## Log4j2 file that configures logging for processes other than Accumulo services
+status = info
+dest = err
+name = AccumuloTestingDefaultLoggingProperties
+
+appender.console.type = Console
+appender.console.name = STDERR
+appender.console.target = SYSTEM_ERR
+appender.console.layout.type = PatternLayout
+# appender.console.layout.pattern = %style{%d{ISO8601}}{dim,cyan} %style{[}{red}%style{%-8c{2}}{dim,blue}%style{]}{red} %highlight{%-5p}%style{:}{red} %m%n
+appender.console.layout.pattern = %d{ISO8601} [%c{3}] %-5p: %m%n
+
+logger.accumulo.name = org.apache.accumulo
+logger.accumulo.level = WARN
+logger.accumulo.testing = DEBUG
```

This isn't quite right. This should be:

```suggestion
logger.accumulotesting.name = org.apache.accumulo.testing
logger.accumulotesting.level = DEBUG
```

The segment after `logger.` is a way of creating a named reference to a logger. Once you have a reference, you need to tell log4j2 what logger name pattern it refers to with `.name = ` and what minimum level to accept for that logger reference with `.level = `.

So, in general:

```ini
logger.refA.name = some.prefix.pattern.here
logger.refA.level = DEBUG
logger.refB.name = some.other.prefix.here
logger.refB.level = WARN
```

In log4j1, you would have put the pattern in the logger reference itself, but you no longer do that in log4j2, since the logger reference is created explicitly with `logger.refName.name = pattern`. So, `logger.accumulo.testing = DEBUG` doesn't do anything, since `.testing` isn't valid.

This error is repeated a few times below. Hopefully this explanation helps.